src/mumble/ConfigWidget.cpp

```diff
@@ -69,3 +69,18 @@ void ConfigWidget::loadComboBox(QComboBox *c, int v) {
 		disconnect(SIGNAL(intSignal(int)));
 	}
 }
+
+void ConfigWidget::loadComboBox(QComboBox *c, const QVariant &v) {
+	const int index = c->findData(v);
+	if (index == -1) {
+		return;
+	}
+
+	if (c->currentIndex() != index) {
+		c->setCurrentIndex(index);
+	} else {
+		connect(this, SIGNAL(intSignal(int)), c, SIGNAL(currentIndexChanged(int)));
```

Comment:
Old signal/slot syntax should no longer be used - prefer the version with explicit function pointers where the compiler can actually do type-checking for us.

Reply:
I only did this for consistency with the rest of the file.

Reply:
I don't think consistency is important for this :shrug: